### DISCUSSION TITLE
umi3 修改配置文件时 如何不重启整个项目 而进行增量编译？

### DISCUSSION THREAD

**Discussion by xakxak007** [umijs/umi#6055] [Q&A]
现在修改一个路由，会自动重启整个项目（大概需要50秒）
有办法增量更新而不重启项目吗？
-------------------------------------------------------------
[change] .umirc.js

INFO Config define changed.
Try to restart dev server...
Starting the development server...

√ Webpack
  Compiled successfully in 51.52s

 DONE  Compiled successfully in 51523ms

**Comment by sorrycc** [comment]
> INFO Config define changed.

这是改了 define 配置，改路由是不会重启的。

**Reply by xakxak007** [reply]
感谢回复！
